src/DynamoFeatureFlags/DynamoFeatureFlags.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <ImportGroup Label="PropertySheets">
+    <Import Project="$(SolutionDir)Config/CS_SDK.props" />
+  </ImportGroup>
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="LaunchDarkly.ClientSdk" Version="2.0.1" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Reference Include="System.Configuration">
+      <HintPath>..\..\..\..\..\..\..\..\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.8\System.Configuration.dll</HintPath>
```

Comment:
not sure whats up with this... we should not need a hint path for a bcl library.

Reply:
indeed, I will look at this:
```
D:\a\Dynamo\Dynamo\Dynamo\src\DynamoFeatureFlags\FeatureFlagsManager.cs(25,30): error CS0103: The name 'ConfigurationManager' does not exist in the current context [D:\a\Dynamo\Dynamo\Dynamo\src\DynamoFeatureFlags\DynamoFeatureFlags.csproj]

    411 Warning(s)
    1 Error(s)
```

Reply:
oh, this is apparently because this functionality requires:
https://www.nuget.org/packages/system.configuration.configurationmanager/
when targeting .net standard 2
@pinzart90 did you run into this anywhere else?

Reply:
also - from my reading it appears I can for the time being move to .net 4.8 target safely, though we'll need to figure this out eventually.

Reply:
System.Configuration was not included in the nestandard list of assemblies/specifications (probably because they wanted to encourage other portable/lightweight solutions - like .json file configs)

They did make it available as a package though (as you mentioned).
I see 3 options here:
1. Use the PackageReference to system.config.manager
    This means that the net48 builds will have the new dll (even though net48 has its own flavor of system.configuration with different APIs)
2. Do multi targeting - net48;net5.0(or 6.0)
    This would require #id #def inside the code to use the different APIs
3. Use a handwritten way (with no external deps)
    Ex. like we do in LibG and [PackageManagerClient](https://github.com/DynamoDS/PackageManagerClient/blob/master/src/GregClient/Utility/Utilities.cs#L44)

Reply:
went with option 3